Text/LanguageDetect.php

```diff
@@ -211,7 +211,7 @@ function _get_data_loc($fname)
 
         } elseif ($this->_data_dir != '@' . 'data_dir' . '@') {
             // if the data dir was set by the PEAR installer, use that
-            return $this->_data_dir . '/Text_LanguageDetect/' . $fname;
+            return $this->_data_dir . '/' . $fname;
```

Comment:
This may work for composer but will break for PEAR. It's probably not ideal for composer as well (will write temp files in the vendor tree).